### DISCUSSION TITLE
PointCloudLayer does not render correctly when using meter_offsets.

### DISCUSSION THREAD

**Discussion by yuby** [visgl/deck.gl#7602] [Q&A] [answered]
HI!

i am using PointCloudLayer with 1milion points data. 
but PointCloudLayer don't work as I expected. 

```
new PointCloudLayer({
      id: 'point-cloud-layer',
      coordinateSystem: COORDINATE_SYSTEM.METER_OFFSETS,
      coordinateOrigin: [originX, originY],
      data,
      pointSize: 2,
      opacity: 0.1,
      visible: true,
      sizeUnits: 'meters'
    });
```

<img width="616" alt="스크린샷 2023-01-27 오후 4 11 27" src="https://user-images.githubusercontent.com/1750850/215115045-6d938184-e037-4de1-b74a-adc0ecadbaba.png">
<img width="657" alt="스크린샷 2023-01-27 오후 4 11 44" src="https://user-images.githubusercontent.com/1750850/215115069-8a180d1b-6d7a-4d5e-bb1a-74a7cbf616f7.png">

it seems deckgl manipulate my data for performance(maybe?). Is there a way to draw it in its original data?

**Comment by Pessimistress** [comment, marked as answer]
`METER_OFFSETS` mode only supports 32-bit precision.

**Reply by yuby** [reply]
thanks @Pessimistress ~

is there better way to render geospatial data with high precision(64bit)? or should i make new custom layer for that? 

please. need help or any advice! thanks!
